app/controllers/campaigns_controller.rb

```diff
@@ -16,6 +16,9 @@ def create
 
   def show
     @campaign = Campaign.find params['id']
+    if @campaign.active == false
+      redirect_to :campaigns, notice: 'The campaign you requested has been deactivated.'
```

Comment:
I think that makes sense. Since we don't currently have an authentication method built in, I'm going to switch this to a 404 for the time being and create a story to allow for campaigners to re-enable a disabled page.